src/java.desktop/unix/classes/sun/awt/X11/XWindowPeer.java

```diff
@@ -804,10 +804,10 @@ public void handleConfigureNotifyEvent(XEvent xev) {
         if (insLog.isLoggable(PlatformLogger.Level.FINE)) {
             insLog.fine(xe.toString());
         }
-        checkIfOnNewScreen(toGlobal(new Rectangle(scaleDown(xe.get_x()),
+        checkIfOnNewScreen(new Rectangle(scaleDown(xe.get_x()),
```

Comment:
After the fix I am unable to create a translucent window on a second screen. (you can reproduce it with your test)

Reply:
Yeah, with [JDK-8006421](https://bugs.openjdk.org/browse/JDK-8006421) fix the transparency issue is resolved. So this bug focus on the correcting the opaque window seen in default (Starting from half of default screen). As highlighted in the below image.
![image](https://github.com/openjdk/jdk/assets/94159358/ebf45077-5308-4e43-a265-2026ef198e11)

Reply:
I see, just the TejeshR13:branch_8280482(from this PR) branch does not contain JDK-8006421 fix. That's why I observed this behavior.